tasks/54_roman-numbers-convert/roman-numbers-convert.js

```diff
@@ -0,0 +1,43 @@
+/**
+* convertToRoman - Converts number from arabic numeral to roman numeral.
+*
+* @example
+* //return 'II'
+* convertToRoman(2)
+* @param {Number} num - number in arabic style.
+* @return {String} equivalent in roman numeral of num.
+*/
+export function convertToRoman(num) {
```

Comment:
It is really hard to understand what's going on here. Could you fix it somehow?

Reply:
@KalachevDev changed some parts for more readable format